### ISSUE TITLE
Code should be more modular

### ISSUE THREAD

**Issue by florianjehn** [allfed/animal-feed-model#2]
Right now everything happens in one single file. However, it is better coding practice to separate things by what they do. Please split at least the model itself and the dash interface in separate files.

**Comment by k-r-a-s-s** [comment]
yep 100% agree, I spoke to @morganrivers about creating classes and generally making this more modular. I'll schedule a call with you to discuss how to do this when you are back next week :)

**Comment by florianjehn** [comment]
Alright :)
You can also take a look at the seaweed growth model. It's not done yet, but this is the direction we want to go to: https://github.com/allfed/Seaweed-Growth-Model
